spec/Listener/CodeCoverageListenerSpec.php

```diff
@@ -35,7 +35,7 @@ public function let(ConsoleIO $io, Driver $driver)
     public function it_can_process_all_directory_filtering_options(SuiteEvent $event)
     {
         $this->setOptions([
-            'blacklist' => [
+            'whitelist' => [
```

Comment:
I think we should add a dedicated `blacklist`-test.

Otherwise our code-coverage will drop.

Sadly the CI pipeline didn't upload the code-coverage to Scrutinizer (https://scrutinizer-ci.com/g/friends-of-phpspec/phpspec-code-coverage/), maybe something is broken there.

Reply:
It has been pushed but you need to check the inspections. The dashboard page is only displaying master build.

Here is the result for the latest push: https://scrutinizer-ci.com/g/friends-of-phpspec/phpspec-code-coverage/inspections/8024bdbd-49db-4a89-91b3-d0a179fdf1e5